modules/gcp/vpc/variables.tf

```diff
@@ -0,0 +1,67 @@
+variable "project_id" {
+  type        = string
+  description = "The ID of the project where this VPC will be created"
+}
+
+variable "region" {
+  type        = string
+  description = "Region in which to deploy the resources"
+}
+
+variable "network_name" {
+  description = "The name of the network being created"
+}
+
+variable "subnets" {
+  type        = list(map(string))
+  default     = []
+  description = "The list of subnets being created"
+}
+
+variable "secondary_ranges" {
+  type        = map(list(object({ range_name = string, ip_cidr_range = string })))
+  description = "Map of secondaryrange in the cluster's subnetwork"
+  default     = {}
+}
```

Comment:
Remove this `secondary_ranges` since it is not vpc specific. `secondary_ranges` should be created by the resources right?